specification/peak-annotation-format/annotation-example-1.json

```diff
@@ -0,0 +1,18 @@
+{
+    "adduct": ["+NH4"],
```

Comment:
Should be renamed to `adducts` as in the schema. There was no conclusion about this during the previous call I think, but I'd advocate to also include the `M` for the adduct, even though it doesn't really give extra information but just to follow standard nomenclature.

Reply:
Agreed to follow standard nomenclature, added `M`.